test/configCases/plugins/environment-plugin/webpack.config.js

```diff
@@ -1,4 +1,4 @@
-const EnvironmentPlugin = require("../../../../lib/EnvironmentPlugin");
+var EnvironmentPlugin = require("../../../../lib/EnvironmentPlugin");
```

Comment:
Do you want me to revert it to const and add use strict or just add use strict`?